_commands/tools/git.md

```diff
@@ -25,6 +25,286 @@ git
 
 ----------------------
 
+Git For Beginners
+=================
+This section of the Git page is for those of us that are actually in EECS 280/281, meaning you have little to no experience with Git and are more worried about figuring out the 
+difference between staging and committing things than the neat little things 
+you can do to make Git as personalized as possible.
+
+What is Git? 
```

Comment:
[Markdown](https://daringfireball.net/projects/markdown/) will also support varying sizes of headers using `#`, `##`, `###`, etc.